firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -383,11 +383,22 @@ private void showUpdateAlertDialog(AppDistributionRelease latestRelease) {
     AlertDialog alertDialog = new AlertDialog.Builder(currentActivity).create();
     alertDialog.setTitle(context.getString(R.string.update_dialog_title));
 
-    alertDialog.setMessage(
-        String.format(
-            "Version %s (%s) is available.",
-            latestRelease.getDisplayVersion(), latestRelease.getBuildVersion()));
+    String message;
+    if (latestRelease.getReleaseNotes().equals("")) {
```

Comment:
Pranav added a method called tryGetValue() in the client that sets fields to empty string if retrieval fails so I don't think it can be null anymore, but I can add a check to be defensive